### ISSUE TITLE
Issue with rendering multiline bulleted list using html

### ISSUE THREAD

**Issue by maltsev-gorskij** [MohamedRejeb/Compose-Rich-Editor#131]
Hi!

I got a problem with rendering bulleted list using html on Android device.

Sample html code 
```
<p style="font-size:16px;line-height: 1.5;">
    <b>Sample bulleted list title:</b>
</p>
<ul>
    <li style="font-size:14px; margin-left: 0px;">Multiline text Multiline text Multiline text Multiline text Multiline text Multiline text</li>
</ul>
```

In Compose multiplatform, it renders with strange padding on the second line of multiline text
![image](https://github.com/MohamedRejeb/Compose-Rich-Editor/assets/55731252/04c21075-5470-4d4a-af6b-0f2544c3ea74)

In Google Chrome, this example renders without second line padding
![image](https://github.com/MohamedRejeb/Compose-Rich-Editor/assets/55731252/b8a4d163-3369-404c-9b78-70a46a332674)

I have also tried [https://compose-richeditor.netlify.app/](https://compose-richeditor.netlify.app/). It offers me to use 
`text-align: left;`, but it did not help. I got the same issue.

Can you please help get rid of this padding or suggest how to do it in your library?
Also, I can't delete padding on the left from marking point so in pushed closer to the left edge of the screen

**Comment by MohamedRejeb** [comment]
Fixed in  `1.0.0-beta05`

**Comment by maltsev-gorskij** [comment]
Thank you!
